CRM/Contact/BAO/Contact.php

```diff
@@ -430,7 +430,7 @@ public static function &create(&$params, $fixAddress = TRUE, $invokeHooks = TRUE
         'is_deceased' => $params['is_deceased'],
         'deceased_date' => $params['deceased_date'] ?? NULL,
       ];
-      CRM_Member_BAO_Membership::updateMembershipStatus($deceasedParams, $params['contact_type']);
+      CRM_Member_BAO_Membership::updateMembershipStatus($deceasedParams);
```

Comment:
There are no other uses of this function in universe, so safe to change signature to remove param no longer needed.